foundry.toml

```diff
@@ -0,0 +1,127 @@
+[profile.default]
+  auto_detect_solc = false
+  bytecode_hash = "none"
+  evm_version = "shanghai"
```

Comment:
Because Cancun does not offer anything that we can benefit from. It introduced blob and transient storage, none of which we use at the moment. 

The upgrade to shanghai was decided in https://github.com/sablier-labs/v2-core/discussions/925 because it introduced `PUSH0` opcode which has gas advantage.